src/marqo/tensor_search/tensor_search.py

```diff
@@ -298,18 +293,14 @@ def get_stats(config: Config, index_name: str):
         vector_count = HttpRequests(config).get(path=f"{index_name}/_search", body=body) \
             ["aggregations"]["nested_chunks"]["marqo_vector_count"]["value"]
     except (KeyError, TypeError) as e:
-        error_message = (f"Marqo received an unexpected response from Marqo-os during execution of `get_stats()` APIs. "
-                         f"The expected fields do not exist in the response. Original error message = {e}")
-        logger.error(error_message)
-        raise errors.InternalError(error_message)
-    except (errors.IndexNotFoundError, errors.InvalidIndexNameError):
-        raise
+        raise errors.InternalError(f"Marqo received an unexpected response from Marqo-OS. "
+                                   f"The expected fields do not exist in the response. Original error message = {e}")
     except errors.MarqoWebError as e:
-        error_message = (
-            f"Marqo encountered an error while communicating with Marqo-os during execution of `get_stats()` APIs. "
-            f"Original error message: {e.message}")
-        logger.error(error_message)
-        raise errors.InternalError(error_message)
+        if isinstance(e, (errors.IndexNotFoundError, errors.InvalidIndexNameError)):
```

Comment:
My bad about this one. Now that I know what you're doing the previous form was perfect